db/schema.rb

```diff
@@ -56,6 +56,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "forecasts", force: :cascade do |t|
+    t.float "lat"
+    t.float "lng"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```

Comment:
This section doesn't belong, because there is no migration file to create it.  You will have to do a db:reset followed by a db:migrate.